src/services/netMgr.js

```diff
@@ -203,17 +222,31 @@ NetMgr.setLocalStorageFromToken = function(token) {
 
 // Add interceptor to detect an expired access_token and refresh;
 NetMgr.axiosInstance.interceptors.response.use(
-    function (response) {
+    (response) => {
+        // If the request was successfully completed set the online status to true.
+        NetMgr.setOnlineStatus(true);
+
         // everything fine! return the response to apiGet/apiPost
         return response;
     },
-    function (error) {
+    (error) => {
         // Get the original request configuration.
         var origCfg = error.config;
 
         // Get the error from that request.
         var origResp = error.response;
 
+        // Grab the error message.
+        var errMsg = error.message || null;
+
+        // Default to assuming we are online.
+        NetMgr.setOnlineStatus(true);
+
+        // Set offline if a Network Error occurs. There should be a more specific error message but I couldn't find one.
+        if(errMsg === 'Network Error') {
```

Comment:
Again, something which needs looking into. I think there's a better thing to compare against.